features/references/manage_retirement.feature

```diff
@@ -0,0 +1,49 @@
+# FeatureID  EpicID StoryID TestID
+Feature: No Hearings after Judicial Office Holder Retirement Date
+
+  As a System Admin
+  I want to be able to set the retirement date of a judicial office holder
+  So that the judicial office holder is not allocated to a hearing beyond the retirement date.
+
+  Background:
+    Given Hearing are setup
```

Comment:
I think it's better to say GIVEN a Hearing for <date> is Created as this is better reuse of an operation that will definitely be needed everywhere.
You also need to state that JOH's are created if you are setting everything up in the background